account/src/use-case/createUserAccount.js

```diff
@@ -1,14 +1,16 @@
 import crypto from 'crypto'
 import { saveAccount } from "../../repositories/accountRepository.js";
+import bcrypt from 'bcryptjs'
 
 export async function createUserUseCase(name, email, password) {
   const createdDate = new Date().toISOString().substring(0, 10);
+  const hashedPassword = bcrypt.hashSync(password, 10);
```

Comment:
I think it's a better approach to put the salt in the .env file or get it from the caller